diff --git a/node_modules/@opennextjs/aws/dist/build/helper.js b/node_modules/@opennextjs/aws/dist/build/helper.js
index 6524181..d77ad5c 100644
--- a/node_modules/@opennextjs/aws/dist/build/helper.js
+++ b/node_modules/@opennextjs/aws/dist/build/helper.js
@@ -374,7 +374,7 @@ export function getPackagePath(options) {
  * @returns the Next.js runtime used: "webpack" or "turbopack"
  */
 export function getBundlerRuntime(options) {
-    const dotNextServerPath = path.join(options.appPath, ".next/server");
+    const dotNextServerPath = path.join(options.appBuildOutputPath, ".next/server");
     if (fs.existsSync(path.join(dotNextServerPath, "webpack-runtime.js"))) {
         return "webpack";
     }
